neuralmonkey/train.py

```diff
@@ -108,18 +116,27 @@ def main() -> None:
            or os.path.exists("{}.0".format(variables_file_prefix))):
         cont_index += 1
 
+        args_file = "{}/args.cont-{}".format(
+            cfg.args.output, cont_index)
         log_file = "{}/experiment.log.cont-{}".format(
             cfg.args.output, cont_index)
         ini_file = "{}/experiment.ini.cont-{}".format(
             cfg.args.output, cont_index)
+        orig_ini_file = "{}/original.ini.cont-{}".format(
+            cfg.args.output, cont_index)
         git_commit_file = "{}/git_commit.cont-{}".format(
             cfg.args.output, cont_index)
         git_diff_file = "{}/git_diff.cont-{}".format(
             cfg.args.output, cont_index)
         variables_file_prefix = "{}/variables.data.cont-{}".format(
             cfg.args.output, cont_index)
 
-    copyfile(sys.argv[1], ini_file)
+    with open(args_file, 'w') as file:
+        print('\n'.join(sys.argv), file=file)
```

Comment:
tak bych to ještě obalil uvozovkama - neni na to nějaká funkce, co to bashovsky oescapuje?